### DISCUSSION TITLE
What version is compatible with targetting .NET Framework 4.0?

### DISCUSSION THREAD

**Discussion by zukarusan** [mysql-net/MySqlConnector#1211] [Q&A] [answered]
I plan to use MySQLConnector in my organization project which is currently targetting to .NET Framework 4.0.

When retrieving this on NuGet package manager I got

```
Executing nuget actions took 150.62 ms
Could not install package 'MySqlConnector 2.1.12'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.0', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.
Time Elapsed: 00:00:00.1884189
========== Finished ==========
```

What least version I can run on my project?

**Comment by bgrainger** [comment, marked as answer]
I don't think MySqlConnector has ever supported .NET Framework 4.0.

.NET 4.0 has been unsupported since 12 January 2016: https://docs.microsoft.com/en-us/lifecycle/faq/dotnet-framework. I know it may not be your decision to make, but I _strongly_ encourage you to update to .NET Framework 4.6.2 or later so that you are using a supported version that's receiving security updates.
